src/pages/StocktakeEditPage.js

```diff
@@ -83,24 +83,23 @@ export class StocktakeEditPage extends React.Component {
   }
 
   renderCell(key, item) {
+    const isEditable = !this.props.stocktake.isFinalised;
```

Comment:
I feel like this variable could be better as isFinalised (just through object destructuring), kind of an extra layer of abstraction but not in a useful way to me. But I assume that you did isEditable as that made more sense to you so I'm not fussed.

Reply:
I think it's easier to understand `isEditable` rather then deduce what is : `isFinalised` and how it applies to the state of cell render.

Reply:
Fair enough.